Traceback (most recent call last):
  File "/home/britta/.local/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/britta/.local/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/britta/.local/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/britta/.local/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/britta/.local/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/britta/.local/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/britta/.local/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
x = np.linspace(0, 2, 1000)

fig, ax = plt.subplots()

fig.subplots_adjust(bottom=0.25)

ax_slider = fig.add_axes([0.2, 0.1, 0.65, 0.03])

def update_lambda(val):
    ax.cla()
    ax.set_ylim(0,5)
    ax.plot(x, expon.pdf(x, scale=1/val), color=default_color)
    ax.set_xlabel('X')
    ax.set_ylabel('P(X)')
    ax.set_title('Exponential distribution')
    fig.canvas.draw_idle()

slider_x = Slider(
    ax = ax_slider,
    label="Î»",
    valmin=0,
    valmax=5,
    valinit=1,
    color=default_color)

slider_x.on_changed(update_lambda)

ax.set_ylim(0,5)
update_lambda(1)
plt.show()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[2], line 24[0m
[1;32m     15[0m     ax[38;5;241m.[39mset_title([38;5;124m'[39m[38;5;124mExponential distribution[39m[38;5;124m'[39m)
[1;32m     16[0m     fig[38;5;241m.[39mcanvas[38;5;241m.[39mdraw_idle()
[1;32m     18[0m slider_x [38;5;241m=[39m Slider(
[1;32m     19[0m     ax [38;5;241m=[39m ax_slider,
[1;32m     20[0m     label[38;5;241m=[39m[38;5;124m"[39m[38;5;124mÎ»[39m[38;5;124m"[39m,
[1;32m     21[0m     valmin[38;5;241m=[39m[38;5;241m0[39m,
[1;32m     22[0m     valmax[38;5;241m=[39m[38;5;241m5[39m,
[1;32m     23[0m     valinit[38;5;241m=[39m[38;5;241m1[39m,
[0;32m---> 24[0m     color[38;5;241m=[39m[43mdefault_color[49m)
[1;32m     26[0m slider_x[38;5;241m.[39mon_changed(update_lambda)
[1;32m     28[0m ax[38;5;241m.[39mset_ylim([38;5;241m0[39m,[38;5;241m5[39m)

[0;31mNameError[0m: name 'default_color' is not defined
NameError: name 'default_color' is not defined

